projects/golang/go/Makefile

```diff
@@ -118,6 +118,18 @@ tar-golang-sources:
 	mkdir -p $(VERSION_DIRECTORY)/archives/$(GOOS)/$(ARCH_LOWER)
 	cd /tmp/go-extracted/usr/lib/golang && rm -rf ./pkg/obj && tar czf $(VERSION_DIRECTORY)/archives/$(GOOS)/$(ARCH_LOWER)/$(GIT_TAG).$(GOOS)-$(ARCH_LOWER).tar.gz .
 
+.PHONY: generate-golang-checksum
+generate-golang-checksum: generate-rpm-checksum generate-archive-checksum
+
+.PHONY: generate-rpm-checksum
+generate-rpm-checksum:
+	mkdir -p $(VERSION_DIRECTORY)/checksums/$(GOOS)/$(ARCH_LOWER)
+	cd /tmp/go-extracted/usr/lib/golang && rm -rf ./pkg/obj && find . \( -not -name . \) -type f -exec cat {} + | sha256sum > $(VERSION_DIRECTORY)/checksums/$(GOOS)/$(ARCH_LOWER)/$(GIT_TAG).$(GOOS)-$(ARCH_LOWER)_RPM_CHECKSUM
```

Comment:
let's output these to the same directory as the corresponding artifact; e.g. if the artifact is in `$(VERSION_DIRECTORY)/rpmbuild/$(RPM_ARCH)`, the checksum will be output to the same directory

Reply:
for the file name, can we just do something like `$FILENAME.sha256`? that way it's really easy to identify the artifact-to-file correspondance and pull down the sha for the file